drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -237,7 +237,10 @@ bridge_added = function(driver, device)
   end
 
   local bridge_ip = bridge_info.ip
-
+  if device:get_field(Fields._REFRESH_AFTER_INIT) == nil then
+    device:set_field(Fields._REFRESH_AFTER_INIT, true, { persist = true })
+  end
+  device:set_field(Fields._REFRESH_AFTER_INIT, true, { persist = true })
```

Comment:
Out of curiosity Doug, if you set Fields._REFRESH_AFTER_INIT after a nil check, why do you set it again? Wouldn't it get set to the same value whether the check returned nil or not? Or is the value getting set different?

Reply:
That is a copy paste bug.